subiquitycore/ui/views/network_configure_interface.py

```diff
@@ -59,18 +61,47 @@ def _build_body(self):
         ])
         return body
 
+    def _set_ipv4_prefixes(self):
+        if len(self.dev.configured_ip_addresses_for_version(4)) > 0:
+            active = 0
+        elif self.dev.dhcp4:
+            active = 1
+        else:
+            active = 2
+        for i in range(len(self.ipv4_method.contents)):
+            b = self.ipv4_method[i]
+            if i == active:
+                p = "(*) "
+            else:
+                p = "( ) "
+            b.set_label(p + b.label[4:])
+
+    def _set_ipv6_prefixes(self):
+        if len(self.dev.configured_ip_addresses_for_version(6)) > 0:
+            active = 0
+        elif self.dev.dhcp6:
+            active = 1
+        else:
+            active = 2
+        for i in range(len(self.ipv6_method.contents)):
+            b = self.ipv6_method[i]
+            if i == active:
+                p = "(*) "
+            else:
+                p = "( ) "
+            b.set_label(p + b.label[4:])
+
     def _build_ipv4_method_buttons(self):
         button_padding = 70
 
         buttons = [
-            menu_btn(label=_("Use a static IPv4 configuration"),
+            menu_btn(label=_("    Use a static IPv4 configuration"),
```

Comment:
This breaks existing translations; better to express this as:
menu_btn(label="    " + _("Use a static IPv4 configuration"),

Reply:
yes! will fix up.